### PR TITLE
Initial support/addition of Teshari species

### PR DESCRIPTION
**Pull request by atakiya**
## About The Pull Request
It's happening

---
Sprites, sounds and other elements by Polaris and VOREstation, with adjustments.

---

#### Base

- [x] Species
- [x] Core body sprites
- [x] Language
- [x] Ears
- [ ] Hairs
- [ ] Tails
- [ ] Markings
- [x] Sounds

#### Clothes

- [ ] Under-clothes
- [ ] Over-clothes
- [ ] Misc accessories
- [ ] Icon alternatives for existing clothes

#### Optional goodies

- [x] Teshari Plushie
- [x] Emotes

## Why It's Good For The Game
More species for people to play with.

## Changelog
:cl: Avunia Takiya
add: Added Tesharii!
add: Added Teshari emotes (chirp, squeak, trill)
imageadd: Added Teshari species sprites
imageadd: Added Teshari plushie sprite
soundadd: Added Teshari sounds (teshchirp, teshsqueak, teshtrill)
config: Added Tesharii as roundstart race
/:cl:

### CODE REVIEW

**Comment by hrzn-actions** [comment]
#### Test Merge Deployed

##### Server Instance
Horizon

##### Revision
Origin: a0a9e6b45172dcf1e96e4bca850d23602d726471
Pull Request: 559fc6252050096aaf05c18ca0ca58a95b1fbeaa
Server: 799cf140a62867b1de6d15f4ba540a73331c116e

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: a0a9e6b45172dcf1e96e4bca850d23602d726471
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: 449939ba3e3d4d6984dd2ff1df5e6d0e88e3e0a1

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: a0a9e6b45172dcf1e96e4bca850d23602d726471
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: f6618b6ac1a67ce4b4147d4879df55bbb3099d2c

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 349b278d84cd3d8a322414997c472956743e5cb7
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: cbe06b66b49bb7c974d2218d88820060bc25be5c

File: hrzn/modules/teshari/code/teshari.dm [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// AVALI BUT FOR SS13 - NOW ON /TG/ CODE - https://avali.fandom.com
+/datum/species/teshari
+	id = "teshari"
+	name = "Teshari"
+	default_color = "0F0"
+	liked_food = GROSS | MEAT | FRIED
+	say_mod = "chirps"
+	species_language_holder = /datum/language_holder/teshari
+	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,HAS_FLESH,HAS_BONE,HAIR,FACEHAIR)
```
Comment (Azarak) [inline hrzn/modules/teshari/code/teshari.dm:9]:
```suggestion
	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,HAS_FLESH,HAS_BONE,NO_UNDERWEAR)
```

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 6659c04c1c6efd3a1471eb3e179435a1618fd3d8
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: e11684af276a99e1512acdfeae4e68394d394b40

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 29dfda67469697557a2f95e66f356ebcdf0630fd
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: 78af1b82f7ce609dffe2c75f5c49017e28890958

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: d787835eb02ad540ea3aa20f66064e6457ea9d0c
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: e76b4ffa5f61205676e578a646fc5033ab630db8

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 7569a1b0f4021bdd89d0e2c629310f24660d995a
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: 3e70ac25f2698033f1d3988cbc993c7debe80291

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 69b1972d302ff0f83658e50d86b223d748462363
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: 41d874ba2aef926e035026857ddb62a97d5b4ddb

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: de36f22bab90280559177ddc44b0a0481938e73b
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: ef605807fa6f462158d6e37f5f69ebf2702b2287

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: de36f22bab90280559177ddc44b0a0481938e73b
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: d54b9befd60b9b3b7f5c8be0bf602b299122ee09

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 8e9dbd7ec9ddf4ab4b58c0c6fd272c2db88421e1
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: 193b2b91b3868f31a92d50d20228ddc16a77f9d5

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 6934b16037441fd212d5b8ab9416dfc009fbc4f2
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: 20ee9983cb98dfd5273e6eec164e358e30853786

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: c458ad007c53fded2d00fc6785e9dc9784e94a48
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: 91ffe2f8a01cc6b7f099521c46e8c930a9cfa080

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: bb300f064b3700b704fdb8d7bed97250d14c5d52
Pull Request: 8f643467f2636d0858efda17ce49f7d586a483d3
Server: ffbc3192cd5addee611c1713a9135dfe6524dea8

File: hrzn/modules/teshari/code/teshari.dm [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	species_language_holder = /datum/language_holder/teshari
+	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,HAS_FLESH,HAS_BONE,NO_UNDERWEAR)
+	inherent_traits = list(TRAIT_RESISTCOLD, TRAIT_ADVANCEDTOOLUSER)
+	attack_verb = "slash"
+	attack_sound = 'sound/weapons/slash.ogg'
+	miss_sound = 'sound/weapons/slashmiss.ogg'
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
+	limbs_icon = 'hrzn/modules/teshari/icons/mob/species/teshari_parts_greyscale.dmi'
+	eyes_icon = 'hrzn/modules/teshari/icons/mob/species/teshari_eyes.dmi'
+	default_mutant_bodyparts = list("tail" = "Teshari Tail")
```
Comment (Azarak) [inline hrzn/modules/teshari/code/teshari.dm:17]:
```suggestion
	default_mutant_bodyparts = list("tail" = "Teshari Tail")
	bodytype = BODYTYPE_TESHARI
```

**Comment by Azarak** [comment]
Outfits would need some sort of handling for special bodytypes. maybe an associative list, keyed by bodytype to another outfit, and then if an outfit would be applied to someone with a bodytype that is included in that list, it'd use the linked outfit instead

File: hrzn/modules/teshari/code/teshari.dm [hunk 3]
```diff
@@ -0,0 +1,33 @@
+// AVALI BUT FOR SS13 - NOW ON /TG/ CODE - https://avali.fandom.com
+/datum/species/teshari
+	id = "teshari"
+	name = "Teshari"
```
Comment (Owai-Seek) [inline hrzn/modules/teshari/code/teshari.dm:4]:
```suggestion
	name = "Teshari"
	flavor_text = "Warning: WIP Race. May not work correctly. ::: A small feathered species, often compared to both birds, and raptors."
```

**Comment by atakiya** [comment]
If someone else wants to pick this up, go ahead.

**Pull request closed without merging**